composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "cron/cron",
+    "name": "iamdevice/cron",
```

Comment:
This can be done using https://getcomposer.org/doc/05-repositories.md#vcs. Will prevent these mistakes :-)